extide/gradle/src/org/netbeans/modules/gradle/api/execute/GradleDistributionManager.java

```diff
@@ -500,7 +499,7 @@ public String toString() {
         })
         public DownloadTask(GradleDistribution dist) {
             this.dist = dist;
-            handle = ProgressHandleFactory.createSystemHandle(Bundle.TIT_Download_Gradle(dist.getVersion()));
+            handle = ProgressHandle.createSystemHandle(Bundle.TIT_Download_Gradle(dist.getVersion()), null);
```

Comment:
I suggest to create a `META-INF/upgrade/*.hint` [Jackpot hint](https://github.com/oracle/graal/blob/master/truffle/src/com.oracle.truffle.api/src/META-INF/upgrade/sourcebuilder.hint) that would recommend this change. It should be straightforward following [this example](https://github.com/oracle/graal/blob/master/truffle/src/com.oracle.truffle.api/src/META-INF/upgrade/sourcebuilder.hint) moreover I believe @jlahoda can help if there are any issues.

Reply:
Tracked as [NETBEANS-5217](https://issues.apache.org/jira/browse/NETBEANS-5217)